README.md

```diff
@@ -1,3 +1,10 @@
 # ImageQuiz
 
 See [the documentation website](http://jasig.github.io/ImageQuiz).
+
+## Discuss
+
++ [imagequiz-discuss Google Group][imagequiz-discuss@]
```

Comment:
interesting I didn't know `@` sign was supported in reference links.
![the more you know](https://i.ytimg.com/vi/GD6qtc2_AQA/maxresdefault.jpg)